modules/update-upgrading-cli.adoc

```diff
@@ -123,11 +121,10 @@ $ oc get clusterversion
 .Example output
 [source,terminal]
 ----
-
 Cluster version is <version>
 
 Upstream is unset, so the cluster will use an appropriate default.
-Channel: stable-4.10 (available channels: candidate-4.10, candidate-4.11, eus-4.10, fast-4.10, fast-4.11, stable-4.10)
+Channel: stable-<version> (available channels: candidate-<version>, eus-<version>, fast-<version>, stable-<version>)
```

Comment:
It's the output of `oc adm upgrade` instead of  `oc get clusterversion` in line https://github.com/openshift/openshift-docs/pull/65618/files#diff-a4543a07ef809868d1b552e88f1c5a5b4dbbc99a3ff5c769ec4ad63cd55d0b4dL120
We can update it to `oc adm upgrade`

Reply:
done :) ty!